README.md

```diff
@@ -138,6 +138,12 @@ Per server settings
     HLL_HOST
     HLL_PORT
     HLL_PASSWORD
+    RCONWEB_SERVER_URL
+
+**You must configure `RCONWEB_SERVER_URL` for each server you're setting up to match the URL you're hosting CRCON on, or you will be unable to access the admin site due to CSRF errors**
```

Comment:
I'm fine with that, but we should definitely point that out in the release announcement :D

Reply:
Definitely.